features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/WSDL/Steps/ProvideWSDL.jsx

```diff
@@ -94,6 +96,39 @@ export default function ProvideWSDL(props) {
     return (
         <React.Fragment>
             <Grid container spacing={5}>
+                <Grid item md={12}>
+                    <FormControl component='fieldset'>
+                        <FormLabel component='legend'>
+                            <React.Fragment>
+                                <sup className={classes.mandatoryStar}>*</sup>{' '}
+                                <FormattedMessage
+                                    id='Apis.Create.WSDL.Steps.ProvideWSDL.implementation.type'
+                                    defaultMessage='Implementation type'
+                                />
+                            </React.Fragment>
+                        </FormLabel>
+                        <RadioGroup
+                            aria-label='Implementation type'
+                            value={apiInputs.type}
+                            onChange={event => inputsDispatcher({ action: 'type', value: event.target.value })}
+                        >
+                            <FormControlLabel
+                                value='PASS'
+                                control={<Radio />}
+                                label={<FormattedMessage id='Apis.Create.WSDL.Steps.ProvideWSDL.passthrough.label' defaultMessage='Pass Through' />}
```

Comment:
Exceeds 120 column limit.

Reply:
Fix in the other places as well. This has caused an eslint failure.

Reply:
Fixed